src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
+        this.line = line;
+        this.comment = comment;
+    }
+
+    public int getLine() {
+        return line;
+    }
+
+    public void setLine(int line) {
+        this.line = line;
+    }
+
+    public String getComment() {
+        return comment;
+    }
+
+    public void setComment(String comment) {
```

Comment:
Similar to 'setLine', directly exposing a setter for 'comment' attribute without validation is a violation of Encapsulation. (Severity: Low)